mmv1/templates/terraform/examples/pubsub_topic_schema_settings.tf.erb

```diff
@@ -0,0 +1,8 @@
+resource "google_pubsub_topic" "<%= ctx[:primary_resource_id] %>" {
```

Comment:
Thanks for adding an example! Do you mind creating an example block similar to the `pubsub_topic_geo_restricted` one in mmv1/products/pubsub/terraform.yaml? This will generate a test and add it to our documentation

Reply:
Thanks for the review! Pushed the changes with the additional example for the schema settings in terraform.yaml